.env

```diff
@@ -3,4 +3,5 @@ BUILD_VARIANT=rcvr
 NEXT_PUBLIC_BUILD_VARIANT=$BUILD_VARIANT
 
 NEXT_PUBLIC_API_BASE=https://api.rcvr.app/
-NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_Q2N3ljTzzBgxO4uOlsv5BS7u00f3eocyfC
\ No newline at end of file
+NEXT_PUBLIC_FRONTEND_URL=https://rcvr.app/
```

Comment:
Shouldn't it be without the trailing slash? The frontend entity in the api has no trailing slash

Reply:
Fixed :)